google/detectors/credentials/ncrack/src/main/java/com/google/tsunami/plugins/detectors/credentials/ncrack/client/NcrackClient.java

```diff
@@ -233,6 +256,9 @@ public TargetService getTargetService() {
     for (NetworkEndpoint networkEndpoint : networkEndpoints) {
       runCommandArgs.add(buildServiceCommandValue(networkEndpoint));
     }
+    if (getMaxTime().length() > 0) {
+        runCommandArgs.add("-g to=" + getMaxTime());
```

Comment:
Please add a unit test for this to verify the command line arg is properly passed into ncrack.

Reply:
unit test has been added. feel free to comment if test coverage needs to be increased.